content/en/docs/Installation/installation-guide/accessing-kiali.md

```diff
@@ -159,10 +187,10 @@ spec:
     web_schema: https
 ```
 
-If you are letting the installation to create an [Ingress resource for you](#ingress-access),
+If you are letting the installation create an [Ingress resource for you](#ingress-access),
```

Comment:
By removing the `to`, isn't this a grammar mistake?
Personally, I find this weird, but I'm not an english speaker :)

Reply:
its actually correct either way. I reads better (to me) that second way :) It is subjective.